dizoo/mujoco/config/walker2d_trex_sac_config.py

```diff
@@ -0,0 +1,105 @@
+from easydict import EasyDict
+
+walker2d_trex_sac_config = dict(
+    exp_name='walker2d_trex_sac_seed0',
+    env=dict(
+        manager=dict(shared_memory=True, reset_inplace=True),
```

Comment:
if both shared_memory=True, reset_inplace=True, this line could be omitted by default.